services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -84,41 +107,41 @@ function updateCaseDetailsWorkflow(keys: CaseKeys, newWorkflow: VivaWorkflow) {
 export async function syncWorkflow(input: LambdaRequest, dependencies: Dependencies) {
   const { personalNumber } = input.detail.user;
 
-  const userCases = await dependencies.getSubmittedOrProcessingOrOngoingCases(personalNumber);
-
-  const caseList = userCases.Items;
-
-  const isEmptyCaseList = caseList?.length === 0;
-
+  const caseList = await dependencies.getCaseList(personalNumber, [
```

Comment:
I think it would be a good idea to rename `getCaseList` to `getCasesByStatusType` here as well. And used like:

```js
dependencies.getCasesByStatusType(personalNumber, [ ... ]);
```